test/functional/FunctionalCodeAction.hs

```diff
@@ -206,7 +210,9 @@ hlintTests = testGroup "hlint suggestions" [
                              , "f = {- inline comment -}{- inline comment inside refactored code -} 1 -- ending comment", ""
                              , "-- final comment"
                              ]
-
+        expectedTypeApp =    [ "module ApplyRefact1 where", ""
+                             , "a = id @Int 1"
+                             ]
```

Comment:
Maybe change these tests to golden tests? If it sounds good, I'm willing to update it.

Reply:
yeah, i was thinking in do it in the past. But otoh i suspect they will not change frequently and to have them close to the code has its value imo.
that said we are using golden tests in other tests and to be consistent is good too, so that change will welcomed, thanks!